tsl/test/t/004_multinode_rdwr_1pc.pl

```diff
@@ -8,12 +8,14 @@
 use warnings;
 use AccessNode;
 use DataNode;
-use TestLib;
 use Test::More tests => 15;
 
 #Initialize all the multi-node instances
 
-my $an = AccessNode->get_new_node('an');
+my $an =
+  ($ENV{PG_VERSION_MAJOR} >= 15)
+  ? AccessNode->new('an')
+  : AccessNode->get_new_node('an');
```

Comment:
You can tweak the `AccessNode` module to introduce the missing function in the `BEGIN` function based on the version, but this is not necessary for this PR.